src/scss/_base.scss

```diff
@@ -36,21 +36,17 @@
 		text-align: left;
 		vertical-align: top;
 	}
-	
+
 	.primary-data {
```

Comment:
Yup, that should be fixed too

On Tue, 31 May 2016 at 12:20 Adam Onishi <notifications@github.com> wrote:

> In src/scss/_base.scss
> <https://github.com/Financial-Times/o-table/pull/31#discussion_r65163919>:
>
> > @@ -36,21 +36,17 @@
> >  		text-align: left;
> >  		vertical-align: top;
> >  	}
> > -	
> > +
> >  	.primary-data {
>
> I know it's not part of the PR but I think updating these to
> o-table__cell--primary style classnames might be better? @j-mes
> <https://github.com/j-mes> was there a reason for making them
> nested/non-BEM? If not I'll open up an issue... / @AlbertoElias
> <https://github.com/AlbertoElias>
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/Financial-Times/o-table/pull/31/files/53af5ab0181ff2ac85b99ad1bf7c93490060954b#r65163919>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA3u3QzQLX21YXQ2uUdIBozpSz3J4R3Mks5qHBmNgaJpZM4IqZql>
> .
>

-- 

------------------------------

*This email was sent by a company owned by Financial Times Group Limited 
("FT Group <http://aboutus.ft.com/corporate-information/#axzz3rajCSIAt>"), 
registered office at Number One Southwark Bridge, London SE1 9HL.  
Registered in England and Wales with company number 879531. This e-mail may 
contain confidential information. If you are not the intended recipient, 
please notify the sender immediately, delete all copies and do not 
distribute it further.  It could also contain personal views which are not 
necessarily those of the FT Group.  We may monitor outgoing or 
incoming emails as permitted by law.*